docker/platypus-deps/Dockerfile

```diff
@@ -134,6 +119,7 @@ WORKDIR /$WORKDIR/mfem/build
 RUN cmake .. \
     -DCMAKE_BUILD_TYPE=Release \
     -DCMAKE_POSITION_INDEPENDENT_CODE=YES \
+    -DMFEM_ENABLE_MINIAPPS=YES \
```

Comment:
I'm not sure how I feel about building all miniapps when we only use one - is this necessary? An issue in the build of any one will lead to this build of MFEM reporting failure (and take longer than necessary) - not sure how much we gain from this.

The `common` miniapp is a bit of an odd one, since it provides utilities that multiple miniapps leverage without them being part of the main framework...

Reply:
That is very true. I thought a bit about it, I left it to gauge your opinion really, e.g. the CSD3 script for the Ampere partition compiles all the miniapps, the Sapphire Rapids one doesn't. I can roll this bit back though. We'll need to run cmake separately from within the `miniapps/common` folder though, but let's leave that for #56.